src/Tizen.Wearable.CircularUI.Forms.Renderer/CirclePageRenderer.cs

```diff
@@ -270,11 +270,20 @@ void UpdateBackground()
         }
         void UpdateActionButton(bool initialize)
         {
+            //remove previous action button
+            if (_actionButton != null)
+            {
+                _actionButton.Clicked -= OnActionButtonClicked;
+                _box.UnPack(_actionButton);
+                _actionButton.Unrealize();
+                _actionButton = null;
+            }
+
             if (Element.ActionButton != null)
             {
                 if (_actionButton == null)
                 {
-                    _actionButton = new ElmSharp.Button(_box)
+                    _actionButton = new Xamarin.Forms.Platform.Tizen.Native.Button(_box)
```

Comment:
Instead of using `Native.Button` just replacing tag characters
``` c#
			_actionButton.Text = Element.ActionButton.Text.Replace("&", "&amp;")
					   .Replace("<", "&lt;")
					   .Replace(">", "&gt;")
					   .Replace(Environment.NewLine, "<br>");
```

Reply:
There will be more need of tag characters. and block furher behavior issue that is not same  as Tizen.Xamarin.Forms.Button.

Reply:
Done. change Tizen.Native.Button to Elmsharp.Button and apply your suggestion code.